GDJS/Runtime/spriteruntimeobject.js

```diff
@@ -4,6 +4,62 @@
  * This project is released under the MIT License.
  */
 
+// Type Definitions
+
+/**
+ * @typedef {Object} Point Represents a Point in a frame
+ * @property {number} x X position of the point
+ * @property {number} y Y position of the point
+ */
+
+/**
+ * @typedef {Object} CustomPointData Represents a Custom Point in a frame
+ * @property {string} name Name of the point (To access it later)
+ * @property {number} x X position of the point
+ * @property {number} y Y position of the point
+ */
+
+/**
+ * @typedef {Object} CenterPointData Represents the center Point in a frame
+ * @property {boolean} automatic Is Automatically found or predefined?
+ * @property {number} x X position of the point
+ * @property {number} y Y position of the point
+ */
+
+/**
+ * @typedef {Object} FrameData Represents a {@link gdjs.SpriteAnimationFrame}
+ * @property {string} [image] The resource name of the image used in this frame
+ * @property {Array<customPointData>} [points] The points of the frame
+ * @property {point} originPoint The origin point
+ * @property {centerPointData} centerPoint The center of the frame
+ * @property {boolean} hasCustomCollisionMask Is The collision mask custom?
+ * @property {Array<Array<point>>} [customCollisionMask] The collision mask if it is custom
+ */
+
+/**
+ * @typedef {Object} DirectionData Represents the data of a {@link gdjs.SpriteAnimationDirection}
+ * @property {number} timeBetweenFrames Time between each frame (FPS)
```

Comment:
These are not FPS but time in seconds:
`Time between each frame, in seconds.`

Reply:
fixed typo